testing/simulator/src/basic_sim.rs

```diff
@@ -81,23 +88,75 @@ pub fn run_basic_sim(matches: &ArgMatches) -> Result<(), String> {
         })
         .collect::<Vec<_>>();
 
-    let mut env = EnvironmentBuilder::minimal()
-        .initialize_logger(LoggerConfig {
-            path: None,
-            debug_level: log_level.clone(),
-            logfile_debug_level: log_level.clone(),
-            log_format: None,
-            logfile_format: None,
-            log_color: false,
-            disable_log_timestamp: false,
-            max_log_size: 0,
-            max_log_number: 0,
-            compression: false,
-            is_restricted: true,
-            sse_logging: false,
-        })?
-        .multi_threaded_tokio_runtime()?
-        .build()?;
+    let logger_config = LoggerConfig {
+        path: None,
+        debug_level: log_level.clone(),
+        logfile_debug_level: log_level.clone(),
+        log_format: None,
+        logfile_format: None,
+        log_color: false,
+        disable_log_timestamp: false,
+        max_log_size: 0,
+        max_log_number: 0,
+        compression: false,
+        is_restricted: true,
+        sse_logging: false,
+    };
+
+    let (env_builder, file_logging_layer, stdout_logging_layer) =
+        EnvironmentBuilder::minimal().init_tracing(logger_config.clone());
+
+    let filter_layer = EnvFilter::try_from_default_env()
+        .or_else(|_| EnvFilter::try_new(log_level.to_lowercase().as_str()))
+        .unwrap();
+
+    let (libp2p_non_blocking_writer, _libp2p_guard, discv5_non_blocking_writer, _discv5_guard) =
+        logging::create_tracing_layer(logger_config.path.clone());
+    let libp2p_layer = tracing_subscriber::fmt::layer()
+        .with_writer(libp2p_non_blocking_writer)
+        .with_line_number(true);
+
+    let discv5_layer = tracing_subscriber::fmt::layer()
+        .with_writer(discv5_non_blocking_writer)
+        .with_line_number(true);
+
+    let sse_logging_layer = match SSE_LOGGING_COMPONENTS.lock() {
+        Ok(guard) => guard.clone(),
+        Err(poisoned) => poisoned.into_inner().clone(),
+    };
```

Comment:
We can also leave out the SSE logging. We just need the most simple form of logging (so long as it matches the regular log output)